MSAL/src/configuration/external/ios/MSALLegacySharedADALAccount.m

```diff
@@ -159,7 +159,7 @@ - (BOOL)matchesParameters:(MSALAccountEnumerationParameters *)parameters
         matchResult &= (self.objectId && [self.objectId caseInsensitiveCompare:parameters.tenantProfileIdentifier] == NSOrderedSame);
     }
     
-    return matchResult &= [super matchesParameters:parameters];
+    return matchResult & [super matchesParameters:parameters];
```

Comment:
Are those two changes produce the same results?

Reply:
Just curious why the old logic is to assign the value then return instead of returning directly :)

Reply:
Yes, it produces the same result. This probably was a copy/paste.